treeherder/webapp/api/performance_data.py

```diff
@@ -64,7 +64,14 @@ def list(self, request, project):
                 framework__in=frameworks)
 
         interval = request.query_params.get('interval')
```

Comment:
I think I'd rather have `start_date` and `end_date` parameters (formatted as 'YYYY-MM-DDTHH:MM:SS'), in addition to `interval`.